static/js/modules/typeahead.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
+
+/* global require, Bloodhound */
+
+var $ = require('jquery');
+require('typeahead.js/dist/typeahead.bundle.js');
```

Comment:
Just `require('typeahead');` doesn't work? I was reading in various issues that it should potentially.